### PR TITLE
Bugfixes

### LINKED ISSUE #1350: Comments sidebar does not remain open when navigating between comments on different pages
**Describe the bug**
After clicking a comment in the comments panel thats on a different page than you are on, you are taken to the page where the comment is located, but the comments panel is not open, the comments are not shown on the page, and your viewport is not fixed on the area of the comment that was clicked on. 

**To Reproduce**
1.  Navigate to a page
2.  open comments panel
3.  click on a comment that is located on another page
4.  page you are taken to does not have comments panel open, nor does it take you to the comments location or show any comments on the canvas

**Expected behavior**
I expect that when I click on the comment in the comments panel on Page A, that when I'm navigated to Page B, the comments panel is open, all of the comments are visible on the canvas, and my viewport is centered on the area of the comment that I had clicked on

**Screenshots**

https://user-images.githubusercontent.com/46250921/142459210-29c5a39d-bec0-4045-9c3d-b1dd9e955e7d.mov

Towards the end of the video I click `c` to make the comments appear because they do not appear on their own

**Desktop (please complete the following information):**

tested on both safari and brave browser

### PR DESCRIPTION
**Pull request by Alotor**
Closes #1350

### CODE REVIEW

File: frontend/src/app/main/ui/components/color_input.cljs [hunk 1]
```diff
@@ -81,7 +94,14 @@
...
 
-        ;; list-id (str "colors-" (uuid/next))
+        on-click
+        (mf/use-callback
+         (fn [event]
+           (let [target (dom/get-target event)]
+             (when (some? ref)
+               (let [current (mf/ref-val ref)]
+                 (when (and (some? current) (not (.contains current target)))
+                   (.blur current)))))))
```
Comment (niwinz) [inline frontend/src/app/main/ui/components/color_input.cljs]: Missing ^js metadata on interop

File: frontend/src/app/main/ui/components/numeric_input.cljs [hunk 2]
```diff
@@ -170,6 +179,15 @@
...
             (when on-blur (on-blur))))
 
+        on-click
+        (mf/use-callback
+         (fn [event]
+           (let [target (dom/get-target event)]
+             (when (some? ref)
+               (let [current (mf/ref-val ref)]
+                 (when (and (some? current) (not (.contains current target)))
+                   (.blur current)))))))
```
Comment (niwinz) [inline frontend/src/app/main/ui/components/numeric_input.cljs]: The same here: ^js

File: frontend/src/app/main/ui/workspace.cljs [hunk 3]
```diff
@@ -87,9 +87,11 @@
 
 (mf/defc workspace-page
   [{:keys [file layout page-id] :as props}]
-  (mf/use-layout-effect
+
+ (mf/use-layout-effect
    (mf/deps page-id)
    (fn []
+     (println ">>" page-id)
```
Comment (niwinz) [inline frontend/src/app/main/ui/workspace.cljs]: :fire:

Comment (Alotor) [reply frontend/src/app/main/ui/workspace.cljs]: ups

File: frontend/src/app/main/ui/components/color_input.cljs (hunk 1 above)
Comment (Alotor) [reply frontend/src/app/main/ui/components/color_input.cljs]: I'll better use the function in dom namespace

File: frontend/src/app/main/ui/components/numeric_input.cljs (hunk 2 above)
Comment (Alotor) [reply frontend/src/app/main/ui/components/numeric_input.cljs]: OK

**Pull request merged**